DOCUMENTATION.md

```diff
@@ -6045,11 +6050,16 @@ balena.models.billing.getAccount(function(error, billingAccount) {
 ```
 <a name="balena.models.billing.getPlan"></a>
 
-##### billing.getPlan() ⇒ <code>Bluebird</code>
+##### billing.getPlan([organization]) ⇒ <code>Bluebird</code>
```

Comment:
Why is it in `[]`?

Reply:
Indeed that's how jsdoc marks optional arguments.
This is also autogenerated and we only specify `@param {(String|Number)} [organization]` in the comments.
PS: You can see the same pattern being used in `application.getAll([options])`.

Reply:
I did a quick search and I didn't find one, but I guess my searching skills failed me :)